### PR TITLE
prepare asciidoc for publication with editorial updates too

### PR DESCRIPTION
**Pull request by amilan17**
DO NOT MERGE!!

This branch is dedicated to the preparation of the ascii docs for WMO publication

### CODE REVIEW

File: standard/requirements/core/REQ_version.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Requirement {counter:req-id}* |*/req/core/version*
-^|A |A WNM SHALL provide information on version conformance via the ``version`` property.
-^|B |The ``version`` property SHALL be fixed to ``v04`` for this version of the specification.
+^|A |A WNM shall provide information on version conformance via the ``version`` property.
+^|B |The ``version`` property shall be fixed to ``v1.0`` for this version of the specification.
 |===
```
Comment (amilan17) [inline standard/requirements/core/REQ_version.adoc:7]: @tomkralidis what is the correct version number that will be used?

File: standard/sections/clause_3_references.adoc [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 * [[ogcapi-features]] OGC: OGC 17-069r, OGC API - Features - Part 1: Core 1.0.1 (2022) footnote:[https://docs.opengeospatial.org/is/17-069r4/17-069r4.html]
 * [[rfc7946]] IETF: RFC-7946 The GeoJSON Format (2016) footnote:[https://datatracker.ietf.org/doc/html/rfc7946]
-* [[rfc8259]] IETF: RFC-8259 The JavaScript Object Notation (JSON) Data Interchange Formathe GeoJSON Format (2016) footnote:[https://datatracker.ietf.org/doc/html/rfc8259]
```
Comment (amilan17) [inline standard/sections/clause_3_references.adoc:5]: fixed typo

File: standard/sections/clause_7_normative_text.adoc [hunk 3]
```diff
@@ -1,19 +1,9 @@
 == The WIS2 Notification Message Encoding
```
Comment (amilan17) [inline standard/sections/clause_7_normative_text.adoc:2]: as discussed, moved text to clause 6

File: standard/sections/clause_3_references.adoc (hunk 2 above)
Comment (tomkralidis) [reply standard/sections/clause_3_references.adoc:5]: Now fixed in https://github.com/wmo-im/wis2-notification-message/commit/edb6593d3cfd268ddfefc854d607594126d0843f

**Pull request closed without merging**